### ISSUE TITLE
Add map preview

### ISSUE THREAD

**Issue by serge1peshcoff** [NoryDev/hitchspots#19]
Another proposal: I think it would be cool if there would be a possibility to display a map preview so the user would be able to see the actual route of the trip.
Not sure though if there would be too much elements after implementing it, so this issue is for the discussion.

I'd be happy to implement it though, I think that this can be slightly useful.

**Comment by NoryDev** [comment]
Yes, I also had this idea, it would be great. With mapbox API + the data in DB we can display a preview of the map with the route and maybe the pins of the spots.

For the visuals, I suggests the following specs:

- mobile first: it should render well and be useful for mobile users (non-obtrusive, scroll should still be natural)
- it's a side information, the main action is the form => probably put the map under the form. Plus don't display anything (not even an empty map until a selection is made in the form.

If you want you can do a POC if you want, go ahead!

**Comment by serge1peshcoff** [comment]
@NoryDev I agree with your point, will try to implement it after making another PR on that repo
